Cargo.toml

```diff
@@ -61,6 +61,10 @@ reqwest = { version = "^0.12", default-features = false, optional = true, featur
 serde = { version = "^1", features = ["derive"], optional = true }
 serde_json = { version = "^1", optional = true }
 url = { version = "^2", optional = true }
+#mime = "0.3.17"
+mime = {git = "https://github.com/PlamenHristov/mime.git", rev = "c30e3db"}
```

Comment:
Wow, just dug for a bit to see why this was coming from your github account, its a shame they won't take any changes until they do a whole philosophical soul-search thing. It looks like the rev you list here is the same one as the tip of main so you should be able to drop that now

Reply:
You mean main of my fork? 

This was just for testing purposes. I was thinking ideally banyan would mirror the changes I've made and use the org repo instead of mine.

Reply:
Yep, main of your fork! But yeah it being in a banyan repo is even better lol